Assets/MixedRealityToolkit.Examples/Demos/Input/Scenes/InputAxesChecker/DisplayInputResult.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Examples.Demos
+{
+    /// <summary>
+    /// Displays a specified axis / button value on a specific TextMesh.
+    /// Will display all active axes and buttons if the input type is None.
+    /// </summary>
+    public class DisplayInputResult : MonoBehaviour
+    {
+        [SerializeField]
+        [Tooltip("Used for displaying data from input.")]
+        private TextMesh displayTextMesh = null;
+
+        [SerializeField]
+        [Tooltip("The type of input to read. Will read all active if set to None.")]
+        private AxisType inputType = AxisType.None;
+
+        [SerializeField]
+        [Tooltip("The axis number to read.")]
+        [Range(1, 28)]
```

Comment:
Do you know if there's any harm to having some other arbitrarily larger numbers here (i.e. as we handle platforms with high axis counts, or as we do things like with that input simulation thingy) it might not be immediately obvious to change this to "unlock" additional functionality

Reply:
Same comment for the button number as well.